gf-guide/prioritization.md

```diff
@@ -0,0 +1,94 @@
+<link href="style.css" rel="stylesheet">
+
+<a href="./index"><button class="button button-i">&larr; GF Guide Index</button></a>
+
+# Prioritization of Adding & Upgrading Fonts to Google Fonts
+{:.no_toc}
+
+<div class="context-reading">
+  
+  Background reading:  
+     <mark class="blue">start</mark> **[Adding & Upgrading Fonts to Google Fonts](./onboarding)**</a>
+     
+</div>
+
+## Table of contents
+{:.no_toc}
+* TOC goes here
+{:toc}
+
+## Priority Definitions
+
+### P0: Top Priority
+
+A large Google Product is blocked from achieving some goal, or soon to be blocked?
```

Comment:
General: excessive use of ? rather than .

Reply:
Fixed in table